api/v2/guides/operator-guide-aws-marketplace.md

```diff
@@ -217,6 +217,9 @@ To create a target group when [creating a load balancer](#creating-a-load-balanc
 
 ## Upgrading UID2 Operator
 
+
+An email notification will be sent to private operators with an update window. After this window, the old version will be deactivated, and operators on this version will stop functioning.
```

Comment:
For each operator version update, private operators receive an email notification with an upgrade window (typically 30 days), after which the old version is deactivated and no longer supported.